packages/arb-node-core/ethbridge/validator.go

```diff
@@ -93,6 +104,27 @@ func (v *ValidatorWallet) executeTransaction(ctx context.Context, tx *types.Tran
 	})
 }
 
+func (v *ValidatorWallet) createWalletIfNeeded(ctx context.Context) error {
+	if v.address == nil {
```

Comment:
Nice to flatten this with an early return

Reply:
I've flattened this somewhat. It's still not perfectly flat, as there might be a case where we've generated an address but don't have a con yet, but it's better than before :)